core/designsystem/src/main/kotlin/com/dhc/designsystem/graph/DhcGraph.kt

```diff
@@ -0,0 +1,288 @@
+                    top.linkTo(graphRowLines.top)
+                    width = Dimension.fillToConstraints
+                    height = Dimension.fillToConstraints
+                }
+        )
+
+        DhcGraphColumnLabel(
+            modifier = Modifier
+                .padding(start = 16.dp)
+                .fillMaxWidth()
+                .constrainAs(graphColumnLabels) {
+                    bottom.linkTo(parent.bottom)
+                    start.linkTo(graphRowLines.start)
+                    end.linkTo(graphRowLines.end)
+                    width = Dimension.fillToConstraints
+                },
+            datas = graphData,
+        )
+    }
+}
+
+@Composable
+private fun DhcGraphRowLines(
+    lines: Int,
+    textHeight: Dp,
+    modifier: Modifier = Modifier,
+) {
+    Column(
+        modifier = modifier,
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+        repeat(lines) {
+            Box(
+                modifier = Modifier.height(textHeight),
+                contentAlignment = Alignment.Center,
+            ) {
+                Box(
+                    modifier = Modifier
+                        .fillMaxWidth()
+                        .height(1.dp)
+                        .background(SurfaceColor.neutral600)
+                )
+            }
+        }
+    }
+}
+
+@Composable
+internal fun DhcGraphRowLabels(
+    labels: List<String>,
+    labelStyle: TextStyle,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier,
+        horizontalAlignment = Alignment.Start,
+        verticalArrangement = Arrangement.SpaceBetween,
+    ) {
+        labels.forEach { label ->
+            Text(
+                text = label,
+                color = SurfaceColor.neutral200,
+                style = labelStyle,
+            )
+        }
+    }
+}
+
+@Composable
+private fun DhcGraphBar(
+    color: Color,
+    modifier: Modifier = Modifier,
+) {
+    Box(
+        modifier = modifier
+            .clip(
+                shape = RoundedCornerShape(
+                    topStart = 8.dp,
+                    topEnd = 8.dp,
+                )
+            )
+            .background(color = color),
+    )
+}
+
+@Composable
+private fun DhcGraphBars(
+    data: List<DhcGraphData>,
+    maxValue: Int,
+    modifier: Modifier = Modifier,
+) {
+    val colors = LocalDhcColors.current
+    val graphWidth = 48.dp
+
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.SpaceAround,
+        verticalAlignment = Alignment.Bottom,
+    ) {
+        repeat(data.size) { index ->
+            Box {
+                DhcGraphBar(
+                    modifier = Modifier
+                        .width(graphWidth)
+                        .fillMaxHeight(data[index].value / maxValue.toFloat()),
+                    color = if (data[index].isHighlight) colors.text.textHighLightsPrimary else SurfaceColor.neutral400,
+                )
```

Comment:
```suggestion
        data.forEachIndexed { index, dhcGraphData ->
            Box {
                DhcGraphBar(
                    modifier = Modifier
                        .width(graphWidth)
                        .fillMaxHeight(dhcGraphData.value / maxValue.toFloat()),
                    color = if (dhcGraphData.isHighlight) colors.text.textHighLightsPrimary else SurfaceColor.neutral400,
                )
```
[Nit]
repeat 의 람다를 통해 전달되는 index 를 사용하는것이라 OutOfBoundException 에 대한 우려는 하지 않아도 되겠지만,, 
`forEachIndexed` 를 사용하면 data[index] 와 같은  get() 과정도 축약할 수 있어서 가독성면에서 더 좋을 것 같아서  `forEachIndexed` 를 제안드립니둥